### ISSUE TITLE
Theatre.z5 not running

### ISSUE THREAD

**Issue by namekuseijin** [curiousdannii/parchment#48]
http://iplayif.com/?story=http%3A%2F%2Fwww.ifarchive.org%2Fif-archive%2Fgames%2Fzcode%2Ftheatre.z5

tried both on Chrome desktop and Chrome android, in 2 different networks.  No messages on javascript console...

running fine on windows frotz.

update:  as noted in discussion, it seems only title screen is not appearing.  Typing space will go through a few blank pages until game starts... so, no game-breaking issue, just some weird presentational change.

**Comment by namekuseijin** [comment]
oh, goodness.  Christminster from about the same time frame, early 90s, is off too:

http://iplayif.com/?story=http%3A%2F%2Fwww.ifarchive.org%2Fif-archive%2Fgames%2Fzcode%2Fminster.z5

I'm pretty sure I ran it on parchment just fine last year...

curses.z5 is ok.

**Comment by jmacdotorg** [comment]
I tried both of these, in Safari.

Can confirm that both seems to load on blank screens. However, Christminster's splash page is hidden after (on my display) several screens' worth of blank lines, so that I have to scroll down quite a bit to see it, but there it is. Theater's initial screen seems to be genuinely blank.

In both cases, hitting any key while the view is scrolled to the bottom causes the screen to clear and the games to begin post-splash-screen, as expected.

**Comment by curiousdannii** [comment]
Hmm, this is an unusual kind of problem. I can't check now, but how many blank lines to these games make on other interpreters?

#9 could help as well.

**Comment by namekuseijin** [comment]
oh, weird!  Suggested by jmac, indeed typing space several times brings on a few more blank pages before actually starting the game, but it does run after all, both of them.

actually, I had previously opened the complaint on a chrome not uptaded in a while and since I had to sign up here, github told me to update the browser and so I did.  Thus, not sure if it was an issue because of new javascript code not being handled well on an old browser.  But anyway, I'm quite sure I did type space quite a few times yesterday on the old browser, to no avail...

so, not an urgent or game-breaking issue, but still not quite the ideal presentation for a game... :)
